sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -764,6 +764,19 @@ class DDLParserSuite extends AnalysisTest {
     assert(exc.getMessage.contains("INSERT INTO ... IF NOT EXISTS"))
   }
 
+  test("update table: columns aliases is not allowed") {
```

Comment:
let's add some simple positive tests as well. You can take a look at how other commands are tested in this suite?

Reply:
Yea, thanks for advising. Will add more cases.